tests/parallel_test_core.py

```diff
@@ -45,7 +56,7 @@ def main():
   cores = int(os.environ.get('EMCC_CORES') or multiprocessing.cpu_count())
   pool = multiprocessing.Pool(processes=cores)
   args = [[x] + sys.argv[1:] for x in optimal_order]
-  filenames = pool.map(run_mode, args, chunksize=1)
+  num_failures = pool.map(run_mode, args, chunksize=1)
```

Comment:
Yeah. The return code number from `tests/runner.py` indicates the actual number of tests that failed, so `0` is a succeeded run, and anything `>0` is a failure, so it's ok to sum them up to get the total number of failed tests.